documentation/source/users/cantherm/input.rst

```diff
@@ -76,6 +76,7 @@ Model Chemistry                                  AEC   BC   SOC
 ``'BMK/cbsb7'``                                   v    v    v
 ``'BMK/6-311G(2d,d,p)'``                          v    v    v
 ``'B3LYP/6-311+G(3df,2p)'``                            v
+``'B3LYP/6-31G**'``                               v
```

Comment:
You're right that I used the other b3lyp bond additivity corrections. I think it's a little ambiguous because I just fit the atomic energies and so what actually is atomic energy vs SOC vs bond additivity is a little cloudy.  I think you're right that this is better though.